docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -342,7 +342,14 @@ The ``query`` function allows you to query the underlying database directly. It
 requires syntax native to PostgreSQL, because the full query is pushed down and
 processed in PostgreSQL. This can be useful for accessing native features which
 are not available in Trino or for improving query performance in situations
-where running a query natively may be faster.
+where running a query natively may be fasters.
+
+.. include:: query-passthrough-warning.fragment
+
+.. Warning::
```

Comment:
Move this all into the fragment

Reply:
I'm trying to find a way. The normal `.. Warning::` syntax won't work inside a .fragment file.